src/wp-includes/post-template.php

```diff
@@ -547,6 +547,21 @@ function get_post_class( $class = '', $post = null ) {
 
 	// All public taxonomies.
 	$taxonomies = get_taxonomies( array( 'public' => true ) );
+
+	/**
+	 * Filters the taxonomies to generate classes for each individual term.
+	 *
+	 * Default is all public taxonomies registered to the post type.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param array    $taxonomies List of all public taxonomies to generate classes for.
+	 * @param string[] $classes    An array of post class names.
+	 * @param string[] $class      An array of additional class names added to the post.
+	 * @param int      $post_id    The post ID.
+	*/
+	$taxonomies = apply_filters( 'post_class_taxonomies', $taxonomies, $classes, $class, $post->ID );
```

Comment:
```suggestion
	$taxonomies = apply_filters( 'wp_post_class_taxonomies', $taxonomies, $classes, $class, $post->ID );
```

namespace all the things: Core hasn't been great at this but it's nice to do for new items.

I'd also consider reordering the parameters:
* taxonomies
* post ID
* classes
* class

Reply:
Thanks for the feedback. Change the order that make more sense.